### ISSUE TITLE
Top Level heading missing

### ISSUE THREAD

**Issue by sharmi** [pythonindia/inpycon2019#71]
A top level heading helps user agents such as screen readers and search engines to know the main topic of a page.

Make sure your page has a top level heading (H1 or WAI-ARIA role="heading" aria-level="1") that matches the main topic of the page.

**Comment by Sayak9495** [comment]
Hi @sharmi, I'm interested. But I've a query, 
Should I add title of the page as the main topic in top-level heading? Or something else?
And should I also add aria level 2 and 3 ?

**Comment by sharmi** [comment]
First, a little background:

   The purpose of this and a few other issues is to make the website
accessible to people with disabilities. Often they use screen readers. When
the webpage has a clearly laid out structure of headings with an initial h1
tag followed by h2 and h3 tags, the screen readers can easily convey the
structure to the user and the user can easily navigate the webpage.

Having that in mind, we would need to have a <h1> tag with content "PyCon
India 2019"

The next levels could be keynote speakers, venues, etc. So they should be
h2 tags.  Please ensure that the formatting doesn't change and we just use
the appropriate tags.

I am sure Siva can provide more pointers on this..

Regards,
Sharmi
